deep_speech_2/examples/librispeech/run_test.sh

```diff
@@ -22,9 +22,9 @@ python -u test.py \
 --num_conv_layers=2 \
```

Comment:
Please also update the default arguments to all other files.

Reply:
Done